### PR TITLE
[fix](nereids) fix unstable mv regression cases

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes
1. trival project cost is changed from 1 to 0
2. analyze mtmv after create view to make case stable
3. add unknown col stats for variant nested column

Issue Number: close #xxx

### CODE REVIEW

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run p0

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run cloud_p0

**Review by hello-stephen — APPROVED** [review]

File: regression-test/data/nereids_p0/expression/fold_constant/fold_constant_by_be.out [hunk 1]
```diff
@@ -2,21 +2,6 @@
...
-PLAN FRAGMENT 0
-  OUTPUT EXPRS:
-    sleep(100)[#0]
-  PARTITION: UNPARTITIONED
-
-  HAS_COLO_PLAN_NODE: false
-
-  VRESULT SINK
-     MYSQL_PROTOCAL
-
```
Comment (seawinde) [inline regression-test/data/nereids_p0/expression/fold_constant/fold_constant_by_be.out:15]: this out result is empty, this is excepted?

File: regression-test/data/mv_p0/k1s2m3/k1s2m3.out [hunk 2]
```diff
@@ -7,41 +7,20 @@
...
-\N	\N
--4	16
-1	1
-2	8
-3	\N
-
--- !select_mv --
-\N	\N
--4	16
-1	1
```
Comment (seawinde) [inline regression-test/data/mv_p0/k1s2m3/k1s2m3.out:27]: this out result is empty, this is excepted?

**Review by starocean999 — APPROVED** [review]

File: regression-test/suites/mv_p0/ut/testAggQueryOnAggMV1/testAggQueryOnAggMV1.groovy [hunk 3]
```diff
@@ -44,84 +44,33 @@ suite ("testAggQueryOnAggMV1") {
...
-    explain {
-        sql("select sum(salary), deptno from emps group by deptno order by deptno;")
-        contains "(emps_mv)"
-    }
-
-    explain {
-        sql("select sum(salary) as salary from emps;")
-        contains "(emps_mv)"
-    }
-
```
Comment (seawinde) [inline regression-test/suites/mv_p0/ut/testAggQueryOnAggMV1/testAggQueryOnAggMV1.groovy:107]: the same with above

File: regression-test/suites/nereids_syntax_p0/mv/ut/distinctQuery.groovy [hunk 4]
```diff
@@ -45,27 +45,9 @@ suite ("distinctQuery") {
...
-        contains "(distinctQuery_mv)"
-    }
-
-    explain {
-        sql("select deptno, count(distinct empid) from distinctQuery group by deptno;")
-        contains "(distinctQuery_mv2)"
-    }
-
-    sql """set enable_stats=true;"""
-
```
Comment (seawinde) [inline regression-test/suites/nereids_syntax_p0/mv/ut/distinctQuery.groovy:61]: the same with avove

File: regression-test/data/nereids_p0/expression/fold_constant/fold_constant_by_be.out (hunk 1 above)
Comment (englefly) [reply regression-test/data/nereids_p0/expression/fold_constant/fold_constant_by_be.out:15]: yes

**Review by morrySnow — APPROVED** [review]

**Pull request merged**